jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/config/JmxScraperConfig.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.config;
+
+import java.util.Collections;
+import java.util.Set;
+
+/** This class keeps application settings */
+public class JmxScraperConfig {
+  String serviceUrl = "";
```

Comment:
[minor] having a static factory allows to make those fields private instead of package private.

Reply:
True, however I still prefer having separate factory. Maybe making the JmxScraperConfigFactory an inner class of JmxScraperConfig would be a good compromise? What do you think? Unfortunately it will make JmxScraperConfig.java quite a big file, which I don't really like...

Reply:
For now let's keep the factory class.